lib/openssl/pkey.rb

```diff
@@ -310,6 +310,12 @@ def to_bn(conversion_form = group.point_conversion_form)
   end
   end
 
+  class PKey
+    def self._load(string)
+      OpenSSL::PKey.read(string)
+    end
+  end
+
```

Comment:
Should this be removed too, as this would be unreachable now?

Reply:
Fixed @ [4d67841](https://github.com/ruby/openssl/pull/646/commits/4d6784194a815128c00f8cab8f699dac06a7f104)